packages/editor/src/store/effects/posts.js

```diff
@@ -103,11 +106,14 @@ export const requestPostUpdate = async ( action, store ) => {
 
 	let request;
 	if ( isAutosave ) {
+		const autosave = dataSelect( 'core' ).getAutosave( post.type, post.id );
+		const mappedAutosave = mapValues( pick( autosave, AUTOSAVE_PROPERTIES ), getPostRawValue );
```

Comment:
Is the `pick()` here necessary? When would `autosave` have a property that's not listed in `AUTOSAVE_PROPERTIES`?